assets/scripts/ui/CloseButton.scss

```diff
@@ -33,8 +33,8 @@ button.close {
 
   // Override font-awesome sizing
```

Comment:
So do we need to still override the sizing now that its a param? 
I'd love a comment here about why were' doing the transform, I can already trust that we're doing transform for good reason but I'm not sure why we're making the icon smaller. :-D

Reply:
Similarly, it's an art direction thing: I preferred the smaller look. It's entirely subjective.

Reply:
Your call, but be aware(and I'm sure you already are) that smaller icons are not as accessible:
https://www.w3.org/WAI/WCAG21/Understanding/target-size.html

44x44 pixels is usually ideal for accessibility. Obviously thats just the ideal, but know that making it that small might be making it harder for some users. Maybe not as much our users, but I've watched some users in user testing really struggle to click on close/delete icons sometimes.

Reply:
Yeah, this is very true. The 40ish px dimension was determined by Apple (I think) and their motivation was driven primarily by making sure people didn't hit the wrong buttons with their fingers on mobile devices. We end up not meeting this guidance in a lot of places, primarily because the UI for Streetmix was not developed for mobile devices. To this day the website is still not accessible on mobile -- literally NOT accessible, we disabled it on purpose -- because it was just not designed with today's audience in mind. It's hard to believe what a difference 7 years makes in how the audience uses the web. This is a weak point for us, and yet, we don't have a plan for implementing a design that really changes how people interact with Streetmix on mobile.

All this to say that I think inflating the button size (without redesigning any of the other UI) raises side effect issues, and it's not the rabbit hole I sought to solve with this icon replacement. If this is the right move for _this_ UI I would likely be considering it. But more to the point I think it would be a small band-aid on what is largely not a mobile-friendly interface.

I'm glad you're thinking about this because we do need to address this problem holistically at some point. I would say this PR is not the time for it.

Reply:
I totally understand. I practically feel contractually obligated to mention accessibility issues since they are easy to overlook. FWIW it is _also_ difficult for people using mouses, it sometimes requires a fine level of motor control that some people do not have. But I understand not wanting to make any further changes at this time.